openverse_catalog/dags/maintenance/pr_review_reminders/pr_review_reminders.py

```diff
@@ -0,0 +1,176 @@
+import datetime
+import logging
+from dataclasses import dataclass
+from typing import Optional
+
+from common.github import GitHubAPI
+
+
+logger = logging.getLogger(__name__)
+
+
+REPOSITORIES = [
+    "openverse",
+    "openverse-catalog",
+    "openverse-api",
+    "openverse-frontend",
+    "openverse-infrastructure",
+]
+
+
+class Urgency:
+    @dataclass
+    class Urgency:
+        label: str
+        days: int
+
+    CRITICAL = Urgency("critical", 1)
+    HIGH = Urgency("high", 2)
+    MEDIUM = Urgency("medium", 4)
+    LOW = Urgency("low", 5)
```

Comment:
It's kind of annoying that you can't reference a class within its own definition. I wish there was something like an enum that also allowed for complex, named values. This is probably the first time I've ever thought to myself "wow I really wish <x> Java feature existed here" :joy: